public/app/features/explore/QueryRow.tsx

```diff
@@ -101,16 +101,23 @@ export class QueryRow extends PureComponent<QueryRowProps, QueryRowState> {
     this.setState({ textEditModeEnabled: !this.state.textEditModeEnabled });
   };
 
-  updateLogsHighlights = debounce((value: DataQuery) => {
```

Comment:
This is the same as on line 16. Not sure why it marked as deleted and then added.🤔